config/config.exs

```diff
@@ -2,4 +2,4 @@
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
-  import_config "./#{Mix.env}.exs"
+import_config "./#{Mix.env()}.exs"
```

Comment:
Obviously unrelated change by `mix format` - I can revert this if preferred.

Reply:
Don't worry about it :)